src/Reports/ElementsInUseReport.php

```diff
@@ -84,14 +80,14 @@ public function columns()
      * Helper method to return the link to edit an element
      *
      * @param string $value
-     * @param ArrayData $item
+     * @param BaseElement $item
      * @return string
      */
     protected function getEditLink($value, $item)
     {
         return sprintf(
             '<a class="grid-field__link" href="%s" title="%s">%s</a>',
-            $item->EditLink,
+            $item->CMSEditLink(),
```

Comment:
Technically this probably violates semver. If anyone is particularly upset about the low likelihood of this change causing issues, I'm OK with putting an if statement in to handle the old ArrayData object too.